.azure-pipelines/templates/build_and_package.yml

```diff
@@ -14,18 +14,24 @@ jobs:
     timeoutInMinutes: 180
     strategy:
       matrix:
-        Linux37:
+        # Linux37:
+        #   vm_image: 'ubuntu-20.04'
+        #   py_version: '3.7'
+        # Linux38:
+        #   vm_image: 'ubuntu-20.04'
+        #   py_version: '3.8'
+        Linux39:
           vm_image: 'ubuntu-20.04'
-          py_version: '3.7'
-        Linux38:
-          vm_image: 'ubuntu-20.04'
-          py_version: '3.8'
-        MacOS37:
-          vm_image: 'macOS-10.14'
-          py_version: '3.7'
-        MacOS38:
+          py_version: '3.9'
+        # MacOS37:
+        #   vm_image: 'macOS-10.14'
+        #   py_version: '3.7'
+        # MacOS38:
+        #   vm_image: 'macOS-10.14'
+        #   py_version: '3.8'
```

Comment:
This commented out code is just temporary to perform a single Python version compatibility check?

Reply:
Yes was just for testing